sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/PowershellManager.java

```diff
@@ -122,7 +122,7 @@ private Mono<Boolean> readData(BufferedReader reader, StringBuilder powerShellOu
     private Mono<Boolean> canRead(BufferedReader reader) {
         Supplier<Boolean> supplier = () -> {
             int pause = 62;
-            int maxPause = Platform.isMac() ? this.waitPause : 500;
```

Comment:
could you only include http proxy changes in this PR and also associate the PR with the customer-reported github issue for better tracking?